src/gui/qgsabstractdatasourcewidget.h

```diff
@@ -60,6 +54,21 @@ class GUI_EXPORT QgsAbstractDataSourceWidget : public QDialog
      */
     void setBrowserModel( QgsBrowserModel *model );
 
+    /**
+     * Returns the dialog map canvas
+     * \see setMapCanvas()
+     *
+     * \since QGIS 3.26
```

Comment:
```suggestion
```

We can remove these, since the method was already present prior to 3.26 (just not virtual, but the docs don't go into that much detail anyway)